web/packages/common/src/helpers/webauthn.ts

```diff
@@ -3,7 +3,7 @@ import * as base64js from "base64-js";
 import { msg } from "@lit/localize";
 
 export function b64enc(buf: Uint8Array): string {
-    return base64js.fromByteArray(buf).replace(/\+/g, "-").replace(/\//g, "_").replace(/=/g, "");
+    return base64js.fromByteArray(buf).replace(/\+/g, "-").replace(/\//g, "_").replace(/[=]/g, "");
```

Comment:
Apparently, `/=/` can be mis-parsed by some tools as an equation, so this is the recommended fix.